e2e/cypress/tests/support/email.ts

```diff
@@ -26,3 +27,24 @@ Cypress.Commands.add('getRecentEmail', ({username, email}) => {
         return cy.wrap({...data, body});
     });
 });
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+
+            /**
+         * getRecentEmail is a task to get an email sent to a user
+         * from the email service provider
+         * @param options.username - username of the user
+         * @param options.email - email of the user
+         *
+         * @example
+         *   cy.getRecentEmail().then((data) => {
+         *       // do something with the email data/content
+         *   });
+         */
+            getRecentEmail(options: Pick<UserProfile, 'username' | 'email'>): Chainable;
+        }
+    }
+}
```

Comment:
Is the idea going to declare type in the same support file like this and then remove the same type originally declared in `.d.ts`? With that, should [cypress/tests/support/email.d.ts](https://github.com/mattermost/mattermost-webapp/blob/b0f817971c5c734fab9a1494e7820b1cf2e4e4ce/e2e/cypress/tests/support/email.d.ts) be removed?

Reply:
Absolutely, thanks for pointing it. It is one of how to instruction